src/main/java/ch/njol/skript/effects/EffReleaseEntityStorage.java

```diff
@@ -0,0 +1,90 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.util.Timespan;
+import ch.njol.skript.util.Timespan.TimePeriod;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.EntityBlockStorage;
+import org.bukkit.entity.Bee;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+@Name("Release Entity Storage")
+@Description({
+	"Release the stored entities in an entity block storage (i.e. beehive).",
+	"Providing a timespan will make the released entities unable to go back into the entity block storage for that amount of time.",
```

Comment:
i'm slightly concerned that this description implies this works for any entity and any storage, instead of just bees and hives